integration/container/resize_test.go

```diff
@@ -0,0 +1,100 @@
+package container
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	req "github.com/docker/docker/integration-cli/request"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/gotestyourself/gotestyourself/poll"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestResizeAPIResponse(t *testing.T) {
+	defer setupTest(t)()
+	client := request.NewAPIClient(t)
+	ctx := context.Background()
+
+	c, err := client.ContainerCreate(ctx,
```

Comment:
We should create a small helper for that (with functors to customize stuff). Because we end up having a lot of common container creation (i.e. busybox with `top`).